chatbot/models.py

```
@@ -0,0 +1,506 @@
+User = get_user_model()
+
+
+class ChatbotSession(models.Model):
+    """
+    جلسه چت‌بات برای پیگیری مکالمات کاربر
+    """
+    
+    SESSION_TYPES = [
+        ('patient', 'بیمار'),
+        ('doctor', 'پزشک'),
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)